mne/viz/_brain/_brain.py

```diff
@@ -246,26 +246,30 @@ def __init__(self, subject_id, hemi, surf, title=None,
                 if not (hemi in ['lh', 'rh'] and h != hemi):
                     ci = hi if hemi == 'split' else 0
                     self._renderer.subplot(ri, ci)
-                    mesh_data = self._renderer.mesh(
-                        x=self.geo[h].coords[:, 0],
-                        y=self.geo[h].coords[:, 1],
-                        z=self.geo[h].coords[:, 2],
-                        triangles=self.geo[h].faces,
-                        color=None,
-                        scalars=self.geo[h].bin_curv,
-                        vmin=geo_kwargs["vmin"],
-                        vmax=geo_kwargs["vmax"],
-                        colormap=geo_kwargs["colormap"],
-                        opacity=alpha,
-                    )
-                    if isinstance(mesh_data, tuple):
-                        actor, mesh = mesh_data
-                        # add metadata to the mesh for picking
-                        mesh._hemi = h
+                    if self._hemi_meshes.get(h) is None:
+                        mesh_data = self._renderer.mesh(
+                            x=self.geo[h].coords[:, 0],
+                            y=self.geo[h].coords[:, 1],
+                            z=self.geo[h].coords[:, 2],
+                            triangles=self.geo[h].faces,
+                            color=None,
+                            scalars=self.geo[h].bin_curv,
+                            vmin=geo_kwargs["vmin"],
+                            vmax=geo_kwargs["vmax"],
+                            colormap=geo_kwargs["colormap"],
+                            opacity=alpha,
+                        )
+                        if isinstance(mesh_data, tuple):
+                            actor, mesh = mesh_data
+                            # add metadata to the mesh for picking
+                            mesh._hemi = h
+                        else:
+                            actor, mesh = mesh_data, None
+                        self._hemi_meshes[h] = mesh
+                        self._hemi_actors[h] = actor
                     else:
-                        actor, mesh = mesh_data, None
-                    self._hemi_meshes[h] = mesh
-                    self._hemi_actors[h] = actor
+                        actor = self._hemi_actors[h]
+                        self._renderer.add_actor(actor)
```

Comment:
Are you allowed in VTK to add the same actor to multiple scenes/views?

Reply:
In practice, it works great for me but apparently this is not recommended...

Reference: http://vtk.1045678.n5.nabble.com/Sharing-vtkRenderer-td5735337.html

I have to rethink all that

Reply:
The example of the thread mentions multiple windows whereas in our case, it's multiple renderers of the same window. I'll rework that anyway

Reply:
Everywhere I look says that actors cannot be shared, the same goes for mappers. The only thing that can be shared is the PolyData itself. I'll restart from scratch.